src/views/admin/applications/AdminHome.tsx

```diff
@@ -0,0 +1,161 @@
+import axios from "axios";
+import { TitleBanner } from "components";
+import Router from "next/router";
+import { useContext, useEffect, useState } from "react";
+import { Col, Row } from "react-bootstrap";
```

Comment:
Chore [repeating]: When [importing components with React Bootstrap](https://react-bootstrap.github.io/getting-started/introduction#importing-components), we should import individual components.